languages/python/sqlalchemy-oso/sqlalchemy_oso/partial.py

```diff
@@ -41,6 +42,13 @@ def flip_op(operator):
     return flips[operator]
 
 
+def add_filter(current, new):
```

Comment:
```suggestion
def and_filter(current, new):
```

Reply:
Added in https://github.com/osohq/oso/pull/861/commits/03715a5e520289fdd0fc344286433a44f981f29a